lib/Utils/TimberDynamicResize.php

```diff
@@ -35,8 +35,6 @@ class TimberDynamicResize
 
     /**
      * Constructor.
-     *
-     * @var TimberDynamicResize
```

Comment:
Stray something. No clue what it wants to be.

Reply:
@szepeviktor: Can confirm this. Thanks!